### ISSUE TITLE
More Features

### ISSUE THREAD

**Issue by ammy20019** [ammy20019/inventory-management-system#1]
**### For this inventory management system we need more enhancement with more features**

It would be great if you could list out some more features that can be added in this project.
Just list out and if possible mention how to approach towards building that feature. 
You can write it down here or just upload a  doc file or a txt file.. anything is fine.

**Comment by deepak725** [comment]
I would like to work on it!

**Comment by ammy20019** [comment]
> 
> 
> I would like to work on it!

Yes Sure! Please go ahead! you can mention your all points in here!!

**Comment by deepak725** [comment]
What features you already have?

**Comment by ammy20019** [comment]
@deepak725  you can checkout in the readme file it's mentioned please go through it!
